### PR TITLE
Fix: #4. RFC references

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

Fixes RFC references

@reschke

### CODE REVIEW

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 1]
```diff
@@ -186,12 +186,12 @@ The goals do not include:
 {::boilerplate bcp14}
 
 This document uses the Augmented BNF defined in {{!RFC5234}} and updated
-by {{!RFC7405}} along with the "#rule" extension defined in Section 7 of
-{{!MESSAGING=RFC7230}}.
+by {{!RFC7405}} along with the "#rule" extension
+defined {{!MESSAGING=RFC7230}}, Section 7.
```
Comment (reschke) [inline draft-ietf-httpapi-ratelimit-headers.md:190]:
```suggestion
defined in Section 7 of {{!MESSAGING=RFC7230}}, .
```

**Review by reschke — CHANGES_REQUESTED** [review]
BTW: if you are running the latest kramdown-rfc2629 you may want to use the builtin support for section references, see https://mailarchive.ietf.org/arch/msg/rfc-markdown/DvG3W4pmeApsE52RVnCDs1M6wsY/

File: draft-ietf-httpapi-ratelimit-headers.md (hunk 1 above)
Comment (ioggstream) [reply draft-ietf-httpapi-ratelimit-headers.md:190]: I'm using Martin's i-d-template docker image, so I'll switch to "in Section xxx of yyy"

**Comment by ioggstream** [comment]
Moved to #20

**Pull request closed without merging**